arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1082,6 +1082,9 @@
             "$ref": "#/definitions/EventHubProperties"
           }
         },
+        "routing": {
```

Comment:
Is adding a new feature to an existing api-version, allowed? Wouldn't this be a breaking change in the sdk?